connections.go

```diff
@@ -50,19 +50,21 @@ type connectionsRoot struct {
 }
 
 type ConnectionPort struct {
-	ID              string             `json:"id"`
-	Name            string             `json:"name,omitempty"`
-	Status          string             `json:"status,omitempty"`
-	Role            ConnectionPortRole `json:"role,omitempty"`
-	Speed           int                `json:"speed,omitempty"`
-	Organization    *Organization      `json:"organization,omitempty"`
-	VirtualCircuits []VirtualCircuit   `json:"virtual_circuits,omitempty"`
-	LinkStatus      string             `json:"link_status,omitempty"`
-	Href            string             `json:"href,omitempty"`
+	Href            string               `json:"href,omitempty"`
```

Comment:
Let's embed the Href type
```suggestion
	*Href         `json:",inline"`
```